src/pages/[lang]/faq/index.tsx

```diff
@@ -246,14 +257,482 @@ const FaqPage: NextPage = () => {
           }}
           className="faqItems"
         >
-          {faqData.sections.map((section, ix) => (
-            <FaqDataSectionBlock
-              key={ix}
-              section={section}
-              setActiveItem={setActiveItem}
-              activeItem={currentItem}
-            />
-          ))}
+          <FaqDataSectionBlock
```

Comment:
can we do something similar to the documents page? have an array of sections with all data and loop through it